tests/test_quickFrameMeasurement.py

```diff
@@ -0,0 +1,112 @@
+#
+# LSST Data Management System
+# Copyright 2008-2017 AURA/LSST.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <https://www.lsstcorp.org/LegalNotices/>.
+
+import os
+
+import unittest
+import numpy as np
+
+import lsst.afw.image as afwImage
+import lsst.utils.tests
+from lsst.pipe.tasks.quickFrameMeasurement import QuickFrameMeasurementTask, QuickFrameMeasurementTaskConfig
+
+
+class QuickFrameMeasurementTaskTestCase(lsst.utils.tests.TestCase):
+    try:
+        afwDataDir = lsst.utils.getPackageDir('afwdata')
```

Comment:
Weird - seems to work without it, and the test _are_ running 🤔
Either `import lsst.utils.tests` is leaking that out so that it works, or some kind of test magic is happening to support that. Either way, I've added the import explicitly now so that it's not fragile if it was the former.